extensions/wikia/LatestPhotos/LatestPhotosHooks.class.php

```diff
@@ -13,7 +13,7 @@ public static function onImageUploadComplete( &$image ){
 		return true;
 	}
 
-	public static function onImageRenameCompleated( &$this , &$ot , &$nt ){
+	public static function onImageRenameCompleated( $this , &$ot , &$nt ){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this use of "$this". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2014)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$this". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$ot". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)